ui/app/routes/jobs/job/index.js

```diff
@@ -9,7 +9,7 @@ export default Route.extend(WithWatchers, {
       return;
     }
     controller.set('watchers', {
-      model: this.watch.perform(model),
+      model: this.watch.perform(model, 2000, controller, 'handleFindRecordError'),
```

Comment:
If this were the true implementation (seems not that likely?), this interface should maybe instead have an `options` object rather than a series of optional parameters.